### PR TITLE
Updated with new content.

### PR DESCRIPTION
**Pull request by jiglei**
Items added:

- Milletian Weapon
- Milletian Armor
- The Boof of the Milletian
- Milletian Focus and Shield
- Red Kitty Necklace (Why not lol)
-105 Earrings (future equipment)

I added extra equipment type to make sure the milletian focus and shield can use the correct scrolls (Truth and Sealed)

Scrolls added:
Wish, Meaningful, Cold-blooded(Future ES)

### CODE REVIEW

File: data.js [hunk 1]
```diff
@@ -26,15 +26,16 @@ var g_charactersLookup = {
...
 }
 
 var g_weaponTypes = {
 	"Fiona":{
 		"longsword":true,
 		"long_hammer":true,
 		"smallshield":true,
-		"largeshield":true
+		"largeshield":true,
+		"milletianshield":true
```
Comment (JamEnergy) [inline data.js]: milletianshield shouldn't be a type (just a name of an item), see below

File: data.js [hunk 2]
```diff
@@ -74,7 +75,8 @@ var g_weaponTypes = {
 	"Arisha":{
 		"spellsword":true,
 		"focus":true,
-		"whip":true
+		"whip":true,
+		"milletianfocus":true
```
Comment (JamEnergy) [inline data.js]: same as above

File: data.js [hunk 3]
```diff
@@ -878,7 +917,56 @@ var g_offhands = [
...
+			"critres": 10,
+			"wil": 80,
+			"agi": 90,
+			"att": 210,
+			"matt": 210
+		}
+	},
+	{
+		"name":"Milletian Focus",
+		"type":"milletianfocus",
```
Comment (JamEnergy) [inline data.js]: type is wrong, the shining focus one was right though

File: data.js [hunk 4]
```diff
@@ -878,7 +917,56 @@ var g_offhands = [
...
+			"critres": 10,
+			"wil": 80,
+			"agi": 90,
+			"matt": 210
+		}
+
+	},
+	{
+		"name":"Milletian Shield",
+		"type":"milletianshield",
```
Comment (JamEnergy) [inline data.js]: type should just be smallshield

**Review by JamEnergy — CHANGES_REQUESTED** [review]
Did you add new types for militian shield/focus to make the scrolls apply on the appropriate offhand items only? That has the correct effect, but I would for now prefer if you didn't create new item types (accepting the "bug" that you can put the new scrolls on old shields).

Could you make that change and re-submit?

I'll add something like a level or name filter to the scroll to fix the resultant bug of the scroll being applicable to items that it isn't meant to be.

Thanks for taking the effort to understand the code and add to the data, though.

**Comment by jiglei** [comment]
I'm sorry if this added categories look ugly, I'll revert it.

I'm not really skilled on this kind of stuff, just wanted to make something functional :)

**Comment by JamEnergy** [comment]
Trailing commas are apparently okay in Javascript, although they are illegal JSON. I don't care enough to ask for that to be changed right now though.

**Pull request merged**